### PR TITLE
[CST-1923] Iterations to the SIT sign-in journey

### PR DESCRIPTION
**Pull request by lazydevorg**
### Context

- Ticket: CST-1923

### Changes proposed in this pull request

* Display the email address we’ve sent the magic link to on sign-in
* Show a redacted email address on the GIAS renominate route
* Confirmation page in the GIAS renomination route

### CODE REVIEW

File: app/views/nominations/request_nomination_invite/review.html.erb [hunk 1]
```diff
@@ -1,19 +1,24 @@
...
 
-        <%= render partial: 'nominations/school_details', locals: { school: @nomination_request_form.school, title: "Your school" }  %>
+    <%= render partial: 'nominations/school_details', locals: { school: @nomination_request_form.school, title: "Your school" } %>
 
-        <%= form_for :confirmation, url: review_request_nomination_invite_path, method: :post do |f| %>
-            <%= f.hidden_field :answers_checked, value: true %>
-            <%= f.govuk_submit "Confirm and send" %>
-        <% end %>
+    <p class="govuk-body">
+      We'll email an access link to
```
Comment (frankieroberto) [inline app/views/nominations/request_nomination_invite/review.html.erb:14]: Typographic apostrophe.

```suggestion
      We’ll email an access link to
```

File: app/views/nominations/request_nomination_invite/success.html.erb [hunk 2]
```diff
@@ -5,9 +5,10 @@
 
     <%= govuk_panel title_text: "Your school has been sent a link", text: nil, classes: "govuk-!-margin-bottom-7" %>
```
Comment (frankieroberto) [inline app/views/nominations/request_nomination_invite/success.html.erb:6]: We’ve removed this panel in favour of a plain h1 - see [page in the prototype](https://manage-training-for-ects-5c9ed99fc0a8.herokuapp.com/check-school-email)
```suggestion
  <h1 class="govuk-heading-xl">Check your school email</h1>
```

File: app/views/nominations/request_nomination_invite/success.html.erb [hunk 3]
```diff
@@ -5,9 +5,10 @@
...
 
-    <p class="govuk-body">We’ve sent a link to the school’s email address at</p>
-
-    <p><strong><%= EmailDecorator::new(@school_email).partial_email %></strong></p>
+    <p class="govuk-body">
+      We’ve sent an email to
+      <strong><%= EmailDecorator::new(@school_email).partial_email %></strong>
+    </p>
 
     <h2 class="govuk-heading-m">What happens next</h2>
```
Comment (frankieroberto) [inline app/views/nominations/request_nomination_invite/success.html.erb:13]: We’ve updated this content, so it's no longer a heading and a list. See [content in the prototype](https://manage-training-for-ects-5c9ed99fc0a8.herokuapp.com/check-school-email).

File: app/views/users/sessions/login_email_sent.html.erb [hunk 4]
```diff
@@ -1,18 +1,17 @@
 <% content_for :title, "Check email" %>
```
Comment (frankieroberto) [inline app/views/users/sessions/login_email_sent.html.erb:1]: This should match the h1 - worth setting a variable and using it in both places maybe? (or use the locale files?)
```suggestion
<% content_for :title, "Check your email" %>
```

File: app/views/users/sessions/login_email_sent.html.erb [hunk 5]
```diff
@@ -1,18 +1,17 @@
...
 
 <div class="govuk-grid-row">
-    <div class="govuk-grid-column-two-thirds">
+  <div class="govuk-grid-column-two-thirds">
 
-        <h1 class="govuk-heading-xl">Check your email</h1>
-        <p class="govuk-body">If you have an account, we’ve sent you a link to sign in.</p>
+    <h1 class="govuk-heading-xl">Check your email</h1>
+    <p class="govuk-body">We've sent an email to <strong><%= @login_email %></strong>.</p>
```
Comment (frankieroberto) [inline app/views/users/sessions/login_email_sent.html.erb:7]: Typographic apostrophe.

```suggestion
    <p class="govuk-body">We’ve sent an email to <strong><%= @login_email %></strong>.</p>
```

File: app/views/users/sessions/login_email_sent.html.erb [hunk 6]
```diff
@@ -1,18 +1,17 @@
...
 
-        <h2 class="govuk-heading-m">If you do not get an email</h2>
-      	<ol class="govuk-list govuk-list--number">
-            <li>Check your spam or junk folder.</li>
-            <li>Try signing in again, making sure your email address is typed correctly and does not include any spaces.</li>
-		</ol>
-        <p class="govuk-body">If you’re not registered, <%= govuk_link_to "find out how to get an account", check_account_path, class: "govuk-link govuk-link--no-visited-state" %>.</p>
-	    <p class="govuk-body">If you still need help, contact us: <%= render MailToSupportComponent.new %></p>
+    <p class="govuk-body">
+      If it does not arrivate within 5 minutes, check your spam or junk folder or
```
Comment (frankieroberto) [inline app/views/users/sessions/login_email_sent.html.erb:10]: Typo.
```suggestion
      If it does not arrive within 5 minutes, check your spam or junk folder or
```

**Review by frankieroberto — CHANGES_REQUESTED** [review]
Looks good! There's a few more content changes to make - I’ve left inline suggestions where I can.

The content on the `/nominations/resend-email` page has also been improved - see [prototype page](https://manage-training-for-ects-5c9ed99fc0a8.herokuapp.com/request-access) (most notably the h1 has changed) - could you also update that one please?

Thanks!

**Comment by frankieroberto** [comment]
Just spotted that this existing error message on the local authority page is a bit odd:

Screenshot 2023-09-11 at 10 26 13

Would it be possible to change it to:

If blank
> Enter your school’s local authority

If the text entered didn't match an option from the autocomplete:
> The local authority must be one from our list

(If this is more than a quick job feel free to leave this for a future follow-up!)

File: app/views/nominations/request_nomination_invite/success.html.erb [hunk 7]
```diff
@@ -3,19 +3,21 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
 
-    <%= govuk_panel title_text: "Your school has been sent a link", text: nil, classes: "govuk-!-margin-bottom-7" %>
+  <h1 class="govuk-heading-xl">Check your school email</h1>
```
Comment (frankieroberto) [inline app/views/nominations/request_nomination_invite/success.html.erb:6]: One minor thing - could you also set the `<title>` tag to the same text as this h1 please? (on line 1)

File: app/views/nominations/request_nomination_invite/resend_email.html.erb [hunk 8]
```diff
@@ -2,15 +2,19 @@
 <% content_for :before_content, govuk_back_link(text: "Back", href: :back) %>
 
 <div class="govuk-grid-row">
-    <div class="govuk-grid-column-two-thirds">
+  <div class="govuk-grid-column-two-thirds">
 
-        <h1 class="govuk-heading-xl">Send your school a link to use this service</h1>
+    <h1 class="govuk-heading-l">Request access to this service</h1>
```
Comment (frankieroberto) [inline app/views/nominations/request_nomination_invite/resend_email.html.erb:7]: One minor thing - could you also set the <title> tag to the same text as this h1 please? (on line 1)

**Review by frankieroberto — CHANGES_REQUESTED** [review]
Thanks for addressing all the feedback! 🙌 

1 last issue which is to make the title tags consistent with the updated h1 headings (sorry I missed this before).

After that we can get this shipped. 🚢 😄

**Review by frankieroberto — APPROVED** [review]

**Review by tonyheadford — APPROVED** [review]

**Pull request merged**